config/prom-k8s/prom-k8s-role.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-k8s
+  namespace: openshift-rbac-permissions
```

Comment:
```suggestion
  namespace: openshift-route-monitor-operator
```

Reply:
oh, I didn't think of it, will parameterize it :)